Ashaba/API-Monitor
# #155450742 Add scheduler collection

#### What does this PR do?
It enables background scheduling
#### Description of Task to be completed?
The front end runs concurrently with background process to check for scheduled checks and runs them.
This updates the database on completion
#### How should this be manually tested?
- Install all the requirements in the `requirements.txt`
- Start redis `redis-server`
- Start the celery beat `celery beat -A celery_worker.celery --loglevel=info`
Ensure to update the config at this stage `CELERY_IMPORTS = ("application.monitoring_tasks",)`
- Start the celery worker `celery worker -A celery_worker.celery --loglevel=info`
- Add a collection and its duration of 1 minute
- Add requests to the collection and wait for the 1 minute.
Apparently the UI is not updated automatically, on refreshing the page, results should show on the dashboard.
#### Any background context you want to provide?
These checks are currently testing only status code and the response time
#### What are the relevant pivotal tracker stories?
155450742
#### Screenshots (if appropriate)
<img width="1230" alt="screen shot 2018-05-02 at 17 21 49" src="https://user-images.githubusercontent.com/15690240/39528656-583f0396-4e2d-11e8-8b1f-8414881a282e.png">
